keps/sig-node/4622-topologymanager-max-allowable-numa-nodes/README.md

```diff
@@ -249,7 +245,7 @@ For GA:
 
 #### GA
 
-- Add a metrics: `kubelet_topology_manager_admission_time`.
+- An existing metric: `kubelet_topology_manager_admission_time` can be used.
```

Comment:
please check `pkg/kubelet/metrics/metrics.go`. I think it is `topology_manager_admission_duration_ms`